pkg/operator/config.go

```diff
@@ -39,6 +48,13 @@ type Images struct {
 	ClusterAPIControllerBareMetal string `json:"clusterAPIControllerBareMetal"`
 	ClusterAPIControllerAzure     string `json:"clusterAPIControllerAzure"`
 	ClusterAPIControllerGCP       string `json:"clusterAPIControllerGCP"`
+	// Images required for the Baremetal pod
+        BaremetalOperator             string `json:"baremetalOperator"`
+        BaremetalProvider             string `json:"baremetalProvider"`
+        BaremetalIronicInspector      string `json:"baremetalIronicInspector"`
+        BaremetalIpaDownloader        string `json:"baremetalIpaDownloader"`
+        BaremetalRhcosDownloader      string `json:"baremetalRhcosDownloader"`
+        BaremetalStaticIpManager      string `json:"baremetalStaticIpManager"`
```

Comment:
appears to be extra indentation on these lines, too

Reply:
There's more of this that I won't comment on, i'd try `hack/go_fmt.sh` to check your formatting.